lib/init.ts

```diff
@@ -13,6 +13,7 @@ export interface InitParams {
   cookieDuration?: number;
   region?: InsightRegion;
   userToken?: string;
+  patch?: boolean;
```

Comment:
A few other ideas that could be more self explanatory: `partial` / `keepSettings`.

Reply:
Other possibilities:

- `preserveOptions`
- `preservePreviousOptions`
- `keepOptions`
- `keepPreviousOptions`
- `mergeOptions`
- `mergeWithPreviousOptions`

cc @Haroenv @benhinchley

Reply:
I like `partial` too: https://github.com/algolia/search-insights.js/pull/401/commits/84872c252f9ca5dfcb12a0fabd47604825a1c13e